images/tkg/docs/examples/adding_os_pkg_repos.md

```diff
@@ -0,0 +1,106 @@
+# Adding new OS packages and configuring the repositories or sources
+
+## Use case
+
+As a user I want to add new OS packages example `pkg1` and `pkg2` and also I want to configure ubuntu/photon sources/repositories to an internal mirror to build the node images in an air-gapped scenario. Configuring new sources/repositories is also useful when we want to install internally built software.
+
+## Customization
+
+Configuration of OS packages and sources/repositories is exposed as packer variables. To view a list of packer variables exposed by [kubernetes image builder][kubernetes-image-builder] please refer to this [page][customizations-doc].
+
+### Adding new packages
+
+To add new packages [kubernetes image builder][kubernetes-image-builder] provides `extra_rpms` and `extra_debs` packer variables for Photon and Ubuntu respectively. To add new packages, add the packages to the packer variables in the [default-args.j2][default-args] file as shown below.
+
+_**Note**: If only building the Ubuntu OS node image then no need to change the Photon OS block._
```

Comment:
_**Note**: If you are building just the Ubuntu OS node image you will not have to modify the photon OS block_